Pocket.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -129,19 +129,19 @@
     {
       "identity" : "swift-collections",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/apple/swift-collections.git",
+      "location" : "https://github.com/apple/swift-collections",
```

Comment:
Is this downgrade expected?

Reply:
uhmmmm not that I'm aware of. Let me look into it

Reply:
> There's some merge conflicts. and should I close this PR? #406

Thanks @timc-mozilla ! Conflicts should be resolved now. Why would you close #406 ?